Assets/Scripts/Gameplay/ConnectionManagement/ConnectionState/ConnectingConnectionState.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using Unity.Multiplayer.Samples.BossRoom.ApplicationLifecycle.Messages;
+using Unity.Multiplayer.Samples.BossRoom.Shared.Infrastructure;
+using Unity.Multiplayer.Samples.Utilities;
+
+namespace Unity.Multiplayer.Samples.BossRoom
+{
+    /// <summary>
+    /// Connection state corresponding to when a client is attempting to connect to a server. If successful,transitions
+    /// to the Connected state. If not, transitions to the Offline state.
+    /// </summary>
+    public class ConnectingConnectionState : ConnectionState
```

Comment:
"ConnectingState"

Reply:
The intent here was to keep the base type as a suffix, but if it makes less sense I will remove it

Reply:
It's really about readability. Right now, I'm reading double connecting, it's weird.